### DISCUSSION TITLE
New in v9.12.0 - ✨ `:CodeCompanionCmd` can write commands for you

### DISCUSSION THREAD

**Discussion by olimorris** [olimorris/codecompanion.nvim#453] [Announcements]
Thanks to @gleachkr for the request and @tizot for the useful references:

https://github.com/user-attachments/assets/c6e5005f-8078-4bb3-8ba6-224072cd4562

**Comment by olimorris** [comment]
Btw, anyone that has a picture of their dog or cat in their profile picture will get enhancement requests bumped up the list 😆.

**Comment by bassamsdata** [comment]
Thank you for adding this feature; I’m sure it’s quite useful. I’ve created a small app script for the command line which is doing the same as copilot feature. 
This feature only works with Copilot, right? I tried switching the adapter to Anthropic and OpenAI, but it didn’t work.

**Reply by olimorris** [reply]
It should work with all adapters now.

**Comment by GitMurf** [comment]
FYI I had no idea how useful this could be until giving it a try! 🚀 Very nice!
